orchagent/vxlanorch.cpp

```diff
@@ -266,6 +317,23 @@ create_tunnel(
         tunnel_attrs.push_back(attr);
     }
 
+    // dest ip
+    if ((dst_ip != nullptr) && p2p)
+    {
+        attr.id = SAI_TUNNEL_ATTR_PEER_MODE;
```

Comment:
With the EVPN feature, the P2P tunnels will be created irrespective of L2 or L3 VXLAN. However the Tunnel Nexthops for the L3VXLAN case use the P2MP tunnel. MAC and IMET point to the created P2P tunnel.

Reply:
@srj102 Since P2P tunnels are used only for L2 EVPN, we could create these tunnel objects only for L2 EVPNs.

Reply:
This would mean the SAI driver has to handle creation of tunnels in the device based on both schemes. Instead creating P2P tunnels also would mean that SAI implementation can create the tunnels in the device based on 
either of the triggers. 

Other aspects like P2P tunnel operstatus, P2P tunnel stats(though not added as part of this release) is useful and 
applicable for L3 VXLAN as well.